packages/salesforce-adapter/test/adapter.crud.test.ts

```diff
@@ -812,6 +813,12 @@ describe('SalesforceAdapter CRUD', () => {
             members: mockDefaultValues.Profile.fullName,
           })
         })
+        it('should return deployment URL containing the deployment ID in the extra properties', async () => {
+          const receivedUrls = result.extraProperties?.deploymentUrls
+          const receivedUrl = _.get(receivedUrls, 0)
```

Comment:
should not use `_.get` when it is not needed, it does much more than necessary and it loses all type information
```suggestion
          const receivedUrl = receivedUrls?.[0]
```

Reply:
v